libs/logging/src/logEventTypes.ts

```diff
@@ -44,6 +45,12 @@ const ApplicationStatusEventDocumentation: LogEventTypeDocumentation = {
     'Status update or message that the application took without user action.',
 };
 
+const ApplicationActionEventDocumentation: LogEventTypeDocumentation = {
+  eventType: LogEventType.ApplicationAction,
+  documentationMessage:
+    'Action taken by the votingworks application automatically when a certain condition is met. Example: When a new USB drive is detected, the application will automatically mount the USB.',
```

Comment:
I think there's a good argument to be made that calling a "USB drive" just "USB" for users is okay since they often call it that. However, I think for our internal stuff we should be careful to distinguish between "USB drives" and "USB in general", i.e. an arbitrary USB device. I'd reword this like so:

```suggestion
    'Action taken by the votingworks application automatically when a certain condition is met. Example: When a new USB drive is detected, the application will automatically mount the it.',
```

Or "the drive" or "the USB drive".